pkg/pdapi/pdapi.go

```diff
@@ -72,6 +75,32 @@ func GetTLSConfig(kubeCli kubernetes.Interface, namespace Namespace, tcName stri
 	return crypto.LoadTlsConfigFromSecret(secret, caCert)
 }
 
+func (pdc *defaultPDControl) GetPDEtcdClient(namespace Namespace, tcName string, tlsEnabled bool) (PDEtcdClient, error) {
+	pdc.mutex.Lock()
```

Comment:
Suggest define separate mutex here, it's an independent client with existing PDClient.

Reply:
updated.